site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -77,23 +116,23 @@ the page that the content script has been [injected][content-inject] into.
 }
 ```
 
-### Getting background data into a content script {: #example-content-msg }
+### Send data from the service worker to a content script {: #example-content-msg }
```

Comment:
I changed this title to make it more explicit.

Reply:
LGTM